File: v131016/TeeNew/Polar_ClockWise.pas
unit Polar_ClockWise;
{$I TeeDefs.inc}

interface

uses
  {$IFNDEF LINUX}
  Windows, Messages,
  {$ENDIF}
  SysUtils, Classes,
  {$IFDEF CLX}
  QGraphics, QControls, QForms, QDialogs, QExtCtrls, QStdCtrls, QComCtrls,
  {$ELSE}
  Graphics, Controls, Forms, Dialogs, ExtCtrls, StdCtrls, ComCtrls,
  {$ENDIF}
  Base, TeEngine, Series, TeePolar, TeeProcs, Chart;

type
  TPolarClockWise = class(TBaseForm)
    Series1: TPolarSeries;
    CheckBox1: TCheckBox;
    procedure CheckBox1Click(Sender: TObject);
    procedure FormCreate(Sender: TObject);
  private
    { Private declarations }
  public
    { Public declarations }
  end;

implementation

{$IFNDEF CLX}
{$R *.DFM}
{$ELSE}
{$R *.xfm}
{$ENDIF}

procedure TPolarClockWise.CheckBox1Click(Sender: TObject);
begin
  Series1.ClockWiseLabels:=CheckBox1.Checked
end;

procedure TPolarClockWise.FormCreate(Sender: TObject);
begin
  inherited;
  Series1.FillSampleValues(20);
end;

initialization
  RegisterClass(TPolarClockWise);
end.

File: TeeNew/Builder2006/Polar_ClockWise.cpp
//---------------------------------------------------------------------------

#include <vcl.h>
#pragma hdrstop

#include "Polar_ClockWise.h"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma link "Base"
#pragma link "TeePolar"
#pragma resource "*.dfm"
TPolarClockWise *PolarClockWise;
//---------------------------------------------------------------------------
__fastcall TPolarClockWise::TPolarClockWise(TComponent* Owner)
        : TBaseForm(Owner)
{
}
//---------------------------------------------------------------------------
void __fastcall TPolarClockWise::CheckBox1Click(TObject *Sender)
{
  Series1->ClockWiseLabels = CheckBox1->Checked;
}
//---------------------------------------------------------------------------

void __fastcall TPolarClockWise::FormCreate(TObject *Sender)
{
  Series1->FillSampleValues(20);
}
//---------------------------------------------------------------------------

